dlp/api/DlpSample/Program.cs

```diff
@@ -0,0 +1,159 @@
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine($"Failed to parse infoType {str}: {e}");
+                    return null;
+                }
+            }).Where(it => it != null);
+        }
+
+        static object InspectString(InspectStringOptions opts)
+        {
+            return InspectLocal(opts, new InspectContentRequest
+            {
+                Parent = $"projects/{opts.ProjectId}",
+                Item = new ContentItem
+                {
+                    Value = opts.Value
+                }
+            });
+        }
+
+        static readonly Dictionary<string, ByteContentItem.Types.BytesType> fileTypes =
+            new Dictionary<string, ByteContentItem.Types.BytesType>()
+        {
+            {"bmp", ByteContentItem.Types.BytesType.ImageBmp},
+            {"jpg", ByteContentItem.Types.BytesType.ImageJpeg},
+            {"jpeg", ByteContentItem.Types.BytesType.ImageJpeg},
+            {"png", ByteContentItem.Types.BytesType.ImagePng},
+            {"svg", ByteContentItem.Types.BytesType.ImageSvg},
+            {"txt", ByteContentItem.Types.BytesType.TextUtf8}
+        };
+
+        static object InspectFile(InspectFileOptions opts)
+        {
+            var fileStream = new FileStream(opts.File, FileMode.Open);
+            try
+            {
+                return InspectLocal(opts, new InspectContentRequest
+                {
+                    Parent = $"projects/{opts.ProjectId}",
+                    Item = new ContentItem
+                    {
+                        ByteItem = new ByteContentItem
+                        {
+                            Data = ByteString.FromStream(fileStream),
+                            Type = fileTypes.GetValueOrDefault(new FileInfo(opts.File).Extension.ToLower(),
+                                    ByteContentItem.Types.BytesType.Unspecified)
+                        }
+                    }
+                });
+            } finally
+            {
+                fileStream.Close();
+            }
+        }
+
+        private static object InspectLocal(InspectLocalOptions opts,
+            InspectContentRequest request)
+        {
+            var inspectConfig = new InspectConfig
+            {
+                MinLikelihood = (Likelihood)opts.MinLikelihood,
+                Limits = new InspectConfig.Types.FindingLimits
+                {
+                    MaxFindingsPerRequest = opts.MaxFindings
+                },
+                IncludeQuote = !opts.NoIncludeQuote
+            };
+            inspectConfig.InfoTypes.AddRange(ParseInfoTypes(opts.InfoTypes));
+            request.InspectConfig = inspectConfig;
+            DlpServiceClient dlp = DlpServiceClient.Create();
+            InspectContentResponse response = dlp.InspectContent(request);
+            var count = 0;
+            var findingsStr = "";
```

Comment:
nit: better string.Empty

Reply:
Now that I see the purpose of this variable - please, convert it to StringBuilder